src/gentropy/config.py

```diff
@@ -123,26 +123,38 @@ class EqtlCatalogueConfig(StepConfig):
 class FinngenStudiesConfig(StepConfig):
     """FinnGen study index step configuration."""
 
+    session: Any = field(
+        default_factory=lambda: {
+            "start_hail": True,
+        }
+    )
     finngen_study_index_out: str = MISSING
+    finngen_phenotype_table_url: str = "https://r11.finngen.fi/api/phenos"
+    finngen_release_prefix: str = "FINNGEN_R11_"
+    finngen_summary_stats_url_prefix: str = (
+        "gs://finngen-public-data-r11/summary_stats/finngen_R11_"
+    )
+    finngen_summary_stats_url_suffix: str = ".gz"
+    efo_curation_mapping_url: str = "https://raw.githubusercontent.com/opentargets/curation/24.09.1/mappings/disease/manual_string.tsv"
```

Comment:
Are these configurations temporary before the pipeline is migrated to orchestration repo, right?